### PR TITLE
Fix label for back button on administrator user page

### PR DESCRIPTION
**Pull request by Lakshan-Banneheke**
### Purpose
Fix label for back button on administrator user page. Currently, the back button displays as "Go back to Users" even when it redirects to the Administrators page.

### Related Issues
- https://github.com/wso2/product-is/issues/17878

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: .changeset/many-forks-exercise.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/console": patch
+"@wso2is/i18n": patch
```
Comment (Achintha444) [inline .changeset/many-forks-exercise.md:3]: bump myaccount and identity-apps-core as well

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 2]
```diff
@@ -12571,7 +12571,7 @@ export const console: ConsoleNS = {
                 title: "Users"
             },
             usersEdit: {
-                backButton: "Go back to Users",
+                backButton: "Go back to {{type}}",
```
Comment (Achintha444) [inline modules/i18n/src/translations/en-US/portals/console.ts:12574]: ned to fix this for french and sinhala as well.

Comment (Lakshan-Banneheke) [reply modules/i18n/src/translations/en-US/portals/console.ts:12574]: Added with [a1e3a33](https://github.com/wso2/identity-apps/pull/4669/commits/a1e3a335219bb95a418df891843749cb2ed944cc)

File: .changeset/many-forks-exercise.md (hunk 1 above)
Comment (Lakshan-Banneheke) [reply .changeset/many-forks-exercise.md]: Done [49e6a46](https://github.com/wso2/identity-apps/pull/4669/commits/49e6a466b17f41cb3bd9d99a1997c40bb0936c1b)

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**